Ginger/Ginger/ApplicationModelsLib/POMModels/POMEditPage.xaml

```diff
@@ -30,30 +33,62 @@
                 </StackPanel>
 
                 <Grid Margin="5,10,5,0" >
+                    <Grid.Resources>
+                        <local:BooleanToVisiblityCollapsedConverter x:Key="visibilityConverter" />
+                    </Grid.Resources>
                     <Grid.ColumnDefinitions>
                         <ColumnDefinition Width="50*"></ColumnDefinition>
                         <ColumnDefinition Width="50*"></ColumnDefinition>
                     </Grid.ColumnDefinitions>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="35" />
+                        <RowDefinition Height="20" />
+                        <RowDefinition Height="Auto" />
+                    </Grid.RowDefinitions>
 
-                    <StackPanel Grid.Column="0" HorizontalAlignment="Left" Orientation="Horizontal">
+                    <StackPanel Grid.Column="0" Grid.Row="0" Margin="5,0,0,0" HorizontalAlignment="Left" Orientation="Horizontal">
                         <Label Style="{StaticResource $LabelStyle}" Content="Target Application:"/>
                         <UserControlsLib1:UCComboBox x:Name="xTargetApplicationComboBox" VerticalAlignment="Top" Width="250"></UserControlsLib1:UCComboBox>
                     </StackPanel>
 
-                    <StackPanel Grid.Column="0" HorizontalAlignment="Left" Orientation="Horizontal">
-                        <Label Content="Page URL:" Style="{StaticResource $LabelStyle}" HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                        <TextBox x:Name="xPageURLTextBox" Style="{StaticResource $TextBoxStyle}" HorizontalAlignment="Left" VerticalAlignment="Center" FontWeight="Bold" TextWrapping="NoWrap" Width="250" Margin="45,0,0,0"/>
-                        <UserControls:ucButton x:Name="xPageURLBtn" ButtonType="ImageButton" HorizontalAlignment="Center" ButtonFontImageSize="16" ButtonImageWidth="16" ButtonImageHeight="16" ButtonImageType="GoNext" Click="xPageURLBtn_Click" ToolTip="Go to Page"/>
-                    </StackPanel>
-
                     <agents:ucAgentControl x:Name="xAgentControlUC" Grid.Column="1" AgentStartedUCEvent="AgentStartedHandler" HorizontalAlignment="Right" Width="350" VerticalAlignment="Top" Margin="0,-5,0,0" ></agents:ucAgentControl>
+
+                    <TextBlock Grid.Column="0" Grid.Row="1" Margin="5,0,0,0" Text="Page Source" FontWeight="Bold" />
```

Comment:
1- for what TextBlock? it should be Label
2- please set it to "Page Load Flow:"

Reply:
Done